watertap/core/zero_order_costing.py

```diff
@@ -665,6 +666,67 @@ def cost_ion_exchange(blk):
         blk.config.flowsheet_costing_block.cost_flow(
             blk.unit_model.resin_demand[t0], "ion_exchange_resin")
 
+    def cost_iron_and_manganese_removal(blk):
+        """
+        General method for costing iron and manganese removal processes. Capital cost
+        is based on the cost of air blower, backwash and dual media filter.
+
+        This method also registers the electricity demand as a costed flow.
+        """
+        t0 = blk.flowsheet().time.first()
+
+        # Get parameter dict from database
+        parameter_dict = \
+            blk.unit_model.config.database.get_unit_operation_parameters(
+                blk.unit_model._tech_type,
+                subtype=blk.unit_model.config.process_subtype)
+
+        # Get costing parameter sub-block for this technology
+        A, B, C, D, E, F = _get_tech_parameters(
+            blk,
+            parameter_dict,
+            blk.unit_model.config.process_subtype,
+            ["capital_blower_a_parameter",
+             "capital_backwash_a_parameter",
+             "capital_backwash_b_parameter",
+             "capital_filter_a_parameter",
+             "capital_filter_b_parameter",
+             "capital_b_parameter"])
```

Comment:
I am not sure this is the best name for this parameter in this case (notably there is no `a` parameter). Maybe `flow_exponent` instead?

Reply:
Revised